README.md

```diff
@@ -0,0 +1,30 @@
+# Vendor
+
+This repo is used for publish packages to npm that are either unreleased or to add things as dependencies instead of resolutions.
```

Comment:
`s/publish/publishing`.

`s/to add things as dependencies instead of resolutions/or depend on unreleased packages. This removes the need for resolutions in users' package.json`